src/main/java/seedu/address/model/person/passenger/Price.java

```diff
@@ -13,16 +12,15 @@
     public static final String MESSAGE_CONSTRAINTS =
             "The price you are willing to pay should be entered as a positive decimal number.";
     public static final String VALIDATION_REGEX = "\\d+(\\.\\d\\d?)?";
```

Comment:
If I'm not wrong, the regex doesn't validate whether the number given is a negative number or not right?
maybe can do `^[1-9]` at start of the pattern to make sure there's no negative sign I guess?

Reply:
Oh nice catch! I'll make the required changes!

Reply:
Nice catch! I didn't think of it but I realised when testing that the \d+ already catches negative cases as "-" is not a digit and so the input string would fail the isValidPrice check!